lib/index.js

```diff
@@ -149,9 +147,9 @@ const create = (customConfig = {}) => {
 
       if (Utils.isHotMode(c)) F.mapObj(entries, Utils.addChunkDevServer)
 
-      /* Generate the Jade template. */
+      /* Generate the Jade rendering function. */
 
-      const template = createTemplate(config.templateExtension)
+      const render = createRenderer(config.jade, config.templateExtension)
```

Comment:
Here we are adding a new config field `jade` to allow the user to customize jade's compile options, right? However I believe this is what the existing option `compile` was for. If I am right about your intention here then I think we should revise this code to be `config.compile` [1].

[1] The field name may be questionable but lets keep that for a different issue so that we can relegate this PR to a bug fix with no interface change.